src/components/SlateEditor/RichTextEditor.jsx

```diff
@@ -101,29 +86,40 @@ const RichTextEditor = class extends React.PureComponent {
       plugins,
       schema,
       value,
-      ...rest
+      renderBlock,
+      renderMark,
+      removeSection,
+      handleSubmit,
+      submitted,
+      index,
     } = this.props;
+
     return (
       <article>
-        <div data-cy="slate-editor" css={slateEditorDivStyle}>
-          <Editor
-            {...classes('content', undefined, className)}
-            {...rest}
-            id={id}
-            onKeyDown={this.onKeyDown}
-            ref={this.editorRef}
-            value={value}
-            name={name}
-            schema={schema}
-            onChange={this.onChange}
-            slateStore={this.state.slateStore}
-            plugins={plugins}
-            placeholder={placeholder}
-            onBlur={onBlur}
-            data-cy={dataCy}
-          />
-          {children}
-        </div>
+        <SlateProvider isSubmitted={submitted}>
+          <div data-cy="slate-editor" css={slateEditorDivStyle}>
+            <Editor
+              {...classes('content', undefined, className)}
+              renderBlock={renderBlock}
+              renderMark={renderMark}
+              removeSection={removeSection}
```

Comment:
Ser forresten at denne brukes i RichTextEditor og trengs ikkje sendes videre til Editor.

Reply:
Det som kansje hadde vert positivt er å lage ts typer for all pluginsene og så importere en union av de hit for å kunne ha oversikt over hva som faktisk trengs

Reply:
tror ikke jeg vil gjøre det i denne omgang da